src/Compilers/CSharp/Portable/Symbols/Source/SourceEventSymbol.cs

```diff
@@ -292,23 +296,27 @@ internal sealed override void DecodeWellKnownAttribute(ref DecodeWellKnownAttrib
             else if (attribute.IsTargetAttribute(this, AttributeDescription.NullableAttribute))
             {
                 // NullableAttribute should not be set explicitly.
+#nullable disable // Can 'arguments.AttributeSyntaxOpt' be null? https://github.com/dotnet/roslyn/issues/39166
```

Comment:
No. This is decoding a source-based attribute: if it is of the attribute type, it's not null.

Reply:
Same for below.

---
In reply to: [364968058](https://github.com/dotnet/roslyn/pull/39194#discussion_r364968058) [](ancestors = 364968058)

Reply:
I'm planning to leave this one for a follow-up

Reply:
Updated both